### PR TITLE
expose clusterclaims by label customized-only

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>
story: https://github.com/open-cluster-management/backlog/issues/12638

### CODE REVIEW

File: pkg/spoke/managedcluster/claim_controller.go [hunk 1]
```diff
@@ -82,7 +85,9 @@ func (c managedClusterClaimController) exposeClaims(ctx context.Context, syncCtx
 	managedCluster *clusterv1.ManagedCluster) error {
 	reservedClaims := []clusterv1.ManagedClusterClaim{}
 	customClaims := []clusterv1.ManagedClusterClaim{}
-	clusterClaims, err := c.claimLister.List(labels.Everything())
+	requirement, _ := labels.NewRequirement(labelCustomizedOnly, selection.DoesNotExist, []string{})
```
Comment (qiujian16) [inline pkg/spoke/managedcluster/claim_controller.go:90]: add a comments here

File: pkg/spoke/managedcluster/claim_controller.go [hunk 2]
```diff
@@ -24,6 +25,8 @@ import (
 	"k8s.io/klog/v2"
 )
 
+const labelCustomizedOnly = "open-cluster-management.io/customized-only"
```
Comment (qiujian16) [inline pkg/spoke/managedcluster/claim_controller.go]: not-sync-to-hub? or spoke-only?

File: pkg/spoke/managedcluster/claim_controller.go (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/spoke/managedcluster/claim_controller.go:90]: done

File: pkg/spoke/managedcluster/claim_controller.go (hunk 2 above)
Comment (zhiweiyin318) [reply pkg/spoke/managedcluster/claim_controller.go]: replace with spoke-only

File: pkg/spoke/managedcluster/claim_controller.go (hunk 1 above)
Comment (qiujian16) [reply pkg/spoke/managedcluster/claim_controller.go:90]: let's say lables of clusterclaim with spoke only will not be synced to managedcluster.Status at hub

Comment (zhiweiyin318) [reply pkg/spoke/managedcluster/claim_controller.go:90]: corrected it.

**Pull request merged**